### PR TITLE
Add Brawler custom skill

### PR DESCRIPTION
**Pull request by darklordabc**

### CODE REVIEW

**Comment by darklordabc** [comment]
@01der use this branch

File: src/game/scripts/vscripts/abilities/brawler_detection.lua [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function brawler_detection( keys )
+	local caster = keys.caster
+	if caster:PassivesDisabled() then return end
```
Comment (DankBudd) [inline src/game/scripts/vscripts/abilities/brawler_detection.lua:3]: @OtsoTurpeinen how is this supposed remove the stats? if you end the script before the buff is removed then they will keep the stats.

Comment (OtsoTurpeinen) [reply src/game/scripts/vscripts/abilities/brawler_detection.lua:3]: I thought I changed that?

**Comment by 01der** [comment]
Thank you! I like it.

**Pull request merged**